app-misc/esh/esh-0.1.1.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="esh evaluates shell expressions in plaintext template files."
+HOMEPAGE="https://github.com/jirutka/esh"
+SRC_URI="https://github.com/jirutka/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="MIT"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~ppc ~ppc64 ~s390 ~sh ~sparc ~x86"
```

Comment:
we should set it to `KEYWORDS="~amd64 ~x86"` until it is really tested once. (see also https://devmanual.gentoo.org/keywording/) this can be **fixed on merge**